Cargo.toml

```diff
@@ -26,7 +26,7 @@ features = ["nightly"]
 travis-ci = { repository = "dalek-cryptography/curve25519-dalek", branch = "master"}
 
 [dev-dependencies]
-sha2 = "0.7"
+sha2 = "0.8"
```

Comment:
Note that by default hash crates now enable `std` feature. It shouldn't matter for tests, but because `std` features are not used, I think it's worth to add `default-features = false`.

Reply:
Updated, although note this is just a dev dependency